docs/platforms/apple/guides/ios/session-replay/customredact.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Using Custom Redact for Session Replay
+sidebar_order: 5501
+notSupported:
+description: "Learn how to redact parts of your app's data in Session Replay."
```

Comment:
```suggestion
description: "Learn how to mask parts of your app's data in Session Replay."
```
i guess we have to adapt the naming here after changing it in the sdk as well :)